### DISCUSSION TITLE
应用包列表不能检测小米系统自带的app

### DISCUSSION THREAD

**Discussion by shui-dun** [MetaCubeX/ClashMetaForAndroid#257] [Q&A] [answered]
设置 > 网络 > 访问控制应用包列表 无法看到小米自带的app，例如应用商店 com.xiaomi.market，而clash for Android可以看到

**Comment by luhongzhen317** [comment]
不只是小米，我这诺基亚手机系统自带的也无法显示。

**Reply by luhongzhen317** [reply, marked as answer]
实际上是可以显示系统应用的，需要在右上角过滤里面把系统应用打上对勾。

**Reply by shui-dun** [reply]
还真是，谢啦

**Comment by shui-dun** [comment]
这样导致小米应用商店下载app会走代理，很麻烦

**Comment by cyfex** [comment]
我现在一个app都看不到，小米手机

**Reply by iwalke** [reply]
那你是没开权限吧。。

**Reply by cyfex** [reply]
呃，我应该没改设置才对，原来可以显示的某天突然不能显示了，也没要我给permission呐

**Comment by rickiey** [comment]
即使勾上系统应用也无法显示google服务等比较底层的应用

**Comment by iwalke** [comment]
一直有一个疑问，为啥手机可以分应用代理？默认模式是像PC的 TUN，接管所有流量？

**Reply by xishang0128** [reply]
安卓基于linux,linux可以使用uid区分流量

**Reply by iwalke** [reply]
那默认是接管所有流量么，像 tun 那样，还是说其他方式？
